bsp/Src/CAN.c

```diff
@@ -448,16 +448,12 @@ can_status_t can_recv(CAN_HandleTypeDef* handle, uint16_t id,
     for (int i = 0; i < can1_recv_entry_count; i++) {
       if (can1_recv_entries[i].id == id) {
         valid_id = true;
-        if (blocking) {
-          // if blocking, retry on empty
-          while (xQueueReceive(can1_recv_entries[i].queue, &payload, 0) ==
-                 errQUEUE_EMPTY) {}
-        } else {
-          // otherwise, finish on empty
-          if (xQueueReceive(can1_recv_entries[i].queue, &payload, 0) ==
-              errQUEUE_EMPTY) {
-            return CAN_EMPTY;
-          }
+
+        // if delay_ticks == portMAX_DELAY thread blocks, 
+        // other values of delay_ticks are delays
```

Comment:
nitpick but I don't like comments explaining the code. I only like comments that outline or explain why something was done